buf.yaml

```diff
@@ -0,0 +1,41 @@
+version: v1
+breaking:
+  use:
+    - FILE
+lint:
+  use:
+    - DEFAULT
+  ignore_only:
+    ENUM_VALUE_PREFIX:
+      - src/proto/dashboard.proto
```

Comment:
The `buf` docs seem to suggest you can just do `src/proto` or even `src` to ignore these instead of specifying all the files here. https://docs.buf.build/lint/configuration#ignore_only

Reply:
Good catch.  These came from `buf lint --error-format=config-ignore-yaml` in order to ignore the existing lint errors.

I suppose there's an open question around whether or now we'd want these rules to apply to new proto files going forward.

Reply:
I've gone ahead and updated the paths for `PACKAGE_DIRECTORY_MATCH`, `PACKAGE_SAME_GO_PACKAGE`, and `PACKAGE_VERSION_SUFFIX` to exclude all files in the proto directory.  But for the others I think there's a case to be made for specific exclusions (we could even do this at the message level instead of for the entire file if we wanted to).